Common/src/main/java/com/tabnineCommon/chat/commandHandlers/chatSettings/UpdateChatSettingsHandler.kt

```diff
@@ -0,0 +1,18 @@
+package com.tabnineCommon.chat.commandHandlers.chatSettings
+
+import com.google.gson.Gson
+import com.google.gson.JsonElement
+import com.intellij.openapi.project.Project
+import com.tabnineCommon.chat.commandHandlers.ChatMessageHandler
+
+class UpdateChatSettingsHandler(gson: Gson) : ChatMessageHandler<ChatSettingsProps, Unit>(gson) {
+    private val chatSettings = ChatSettings(gson)
+
+    override fun handle(payload: ChatSettingsProps?, project: Project) {
+        payload?.let { chatSettings.save(it) }
```

Comment:
warn-log or something if the payload is not present - it should be present, and if not the state will not be set properly and we won't know why

Reply:
I don't see we log this kind of warning on the other handlers. is this necessary? bc with the correct types, the payload should exist in this command.